docs: add a basic layout for wait strategies in docs (#536)
testcontainers/testcontainers-go: docs/features/wait/health.md

# Health Wait strategy

> The health wait strategy will check the container is in the healthy state, being able to set the following conditions:

Reviewer: ```suggestion
The health wait strategy will check that the container is in the healthy state and allows to set the following conditions:
```

> - the startupTimeout to be used, default is 60 seconds.

Reviewer: ```suggestion
- the startup timeout to be used in seconds, default is 60 seconds.
```
Since `exit timeout` hat a space, I thought this is more consistent.

> - the PollInterval to be used, default is 100 milliseconds.

Reviewer: ```suggestion
- the poll interval to be used in milliseconds, default is 100 milliseconds.
```
